File: day2/main.py
def check_access() -> str:
    """Ask the user for their role."""
    
    role = input("Enter your role: ")

    if role.lower() == "admin":
        return print("Full Access Granted")
    elif role.lower() == "manager":
        return print("Limited Access Granted")
    else:
        return print("Access Denied")

def main():
    """execute main program"""
    check_access()


if __name__ == "__main__":
    main()

File: day2/main.go
package main

import (
	"fmt"
	"strings"
)

func checkAccess() {
	var role string
	fmt.Print("Enter your role: ")
	fmt.Scan(&role)
	
	switch strings.ToLower(role) {
	case "admin":
		fmt.Print("Full Access Granted\n")
	case "manager":
		fmt.Print("Limited Access Granted\n")
	default:
		fmt.Print("Access Denied\n")
	}
}

func main() {
	checkAccess()
}